recipe/meta.yaml

```diff
@@ -51,6 +49,7 @@ test:
     - pytest-cov
     - pip
     - importlib_metadata
+    - crc32c >=2.7
```

Comment:
Submitted upstream PR ( https://github.com/zarr-developers/numcodecs/pull/635 ) so these tests are skipped gracefully if `crc32` is not present

Reply:
On further exploration, it looks like `crc32c` support is not being handled like other soft dependencies

To clean this up a bit, submitted PR: https://github.com/zarr-developers/numcodecs/pull/637